<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormManageReportings</class>
 <widget class="QWidget" name="FormManageReportings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxReportings">
     <property name="title">
      <string>Gestion des reportings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="comboListOfReportings"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeleteReporting">
        <property name="text">
         <string>Supprimer reporting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nouveau reporting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboListOfTrees"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDisplayTree">
        <property name="text">
         <string>Afficher l'arbre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddReporting">
        <property name="text">
         <string>Nouveau reporting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
